src/planner/plan/append_simple.cpp

```diff
@@ -40,6 +46,26 @@ void Planner::appendCreateTable(const BoundStatement& statement, LogicalPlan& pl
     auto op = make_shared<LogicalCreateTable>(info->copy(),
         statement.getStatementResult()->getSingleColumnExpr());
     plan.setLastOperator(std::move(op));
+    // If it is CREATE NODE TABLE AS, then copy as well
+    if (createTable.hasCopyInfo()) {
+        const auto copyInfo = createTable.getCopyInfo();
+        const auto catalog = clientContext->getCatalog();
+        const auto transaction = clientContext->getTransaction();
+        if (catalog->containsTable(transaction, info->tableName)) {
```

Comment:
This is not the right place to handle NOT EXISTS. Remove and open an issue about supporting CREATE TABLE AS with IF (NOT) EXISTS

Reply:
Removed with https://github.com/kuzudb/kuzu/pull/5354/commits/1a58c6f3626e5511b2ea87c3b3afb6c1372c360a and opened the issue https://github.com/kuzudb/kuzu/issues/5406.